### PR TITLE
ascanrules: Cloud Metadata Potentially Exposed AWS/GCP IP Fix

### PR DESCRIPTION
**Pull request by kingthorin**
- CHANGELOG > Add fix note.
- CloudMetadataScanRule > Correct IP.
- CloudMetadataScanRuleUnitTest > Correct IP.

Fixes zaproxy/zaproxy#7829

### CODE REVIEW

File: addOns/ascanrules/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Maintenance changes.
 
+### Fixed
+- Correct IP used for AWS (GCP) in the Cloud Metadata Potentially Exposed scan rule (Issue 7829).
```
Comment (thc202) [inline addOns/ascanrules/CHANGELOG.md:11]: Why is GCP in parentheses?

Comment (kingthorin) [reply addOns/ascanrules/CHANGELOG.md:11]: Because it also happens to apply for Google

Comment (kingthorin) [reply addOns/ascanrules/CHANGELOG.md:11]: Oh I see we do mention GCP in alerts even if not code, I’ll tweak it.

**Comment by kingthorin** [comment]
Tweaked.

**Review by ricekot — APPROVED** [review]
Thanks!

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**